hadoop-ozone/common/src/test/java/org/apache/hadoop/ozone/om/lock/TestOzoneManagerLock.java

```diff
@@ -426,8 +444,58 @@ public void testReadLockConcurrentStats() throws InterruptedException {
 
       String readHeldStat = lock.getReadLockHeldTimeMsStat();
       Assert.assertTrue(
-          "Expected " + threadCount + " samples in " + readHeldStat,
+          "Expected " + threadCount +
+              " samples in readLockHeldTimeMsStat: " + readHeldStat,
           readHeldStat.contains("Samples = " + threadCount));
+
+      String readWaitingStat = lock.getReadLockWaitingTimeMsStat();
+      Assert.assertTrue(
+          "Expected " + threadCount +
+              " samples in readLockWaitingTimeMsStat: " + readWaitingStat,
+          readWaitingStat.contains("Samples = " + threadCount));
+    }
+  }
+
+  @Test
+  public void testWriteLockConcurrentStats() throws InterruptedException {
```

Comment:
@tanvipenumudy Can you add a test case for mix of (synthetic) readers/writers. Here you can have more readers than writers. For ex: 3 writers and 10 readers.

`public void testSyntheticReadWriteLockConcurrentStats()`

Reply:
Thank you for the comment, added `testSyntheticReadWriteLockConcurrentStats()` to produce a mix of multiple readers + writers.